shared/templates/platform_os_linux/ansible.template

```diff
@@ -0,0 +1,12 @@
+{{%- set ansible_os_release_name_cond = 'ansible_distribution == "' + OS_ID_ANSIBLE + '"' -%}}
+{{%- macro ansible_os_release_conditional(specs) -%}}
+  {{%- for spec in specs -%}}
+    ansible_distribution_version is version("{{{ spec.ver }}}", "{{{ spec.op }}}")
```

Comment:
I assume that this means that this PR depends on the versions feature introduced in #9945. Is that correct?

Reply:
No, sorry for confusion. I force pushed the last commit and removed version support from Ansible remediation.